### DISCUSSION TITLE
[useRangeCalendar] Disable selecting End Date on hover

### DISCUSSION THREAD

**Discussion by knoefel** [adobe/react-spectrum#4577] [Q&A]
I am seeking to integrate the selection of a range and an individual date within the same calendar component.

I wonder if it's possible to deactivate the automatic end date selection while hovering over subsequent dates, following the selection of a start date. Ideally, I would prefer to allow end date selection only when a user actively clicks on a date cell.

Also, is there a way to configure the `useRangeCalendar` hook so that when a user picks a start date with the keyboard, it doesn't automatically go to the next day? I'd like it to stay on the start date cell that was just picked (also for the single date selection use case).

I appreciate any feedback, thank you in advance!

**Comment by LFDanLu** [comment]
> I wonder if it's possible to deactivate the automatic end date selection while hovering over subsequent dates, following the selection of a start date. Ideally, I would prefer to allow end date selection only when a user actively clicks on a date cell.

The date range selection shouldn't be confirmed until the user clicks on a separate date already, do you have an example showing this issue? 

> Also, is there a way to configure the useRangeCalendar hook so that when a user picks a start date with the keyboard, it doesn't automatically go to the next day? I'd like it to stay on the start date cell that was just picked (also for the single date selection use case).

We felt it would be valuable to automatically go to the next day when using keyboard selection so it was clear to the user that a range selection has been started, so at the moment there isn't a way to configure that behavior.
